acq4/devices/PatchPipette/statemanager.py

```diff
@@ -139,7 +139,7 @@ def configureState(self, state, *args, **kwds):
                 self.configureState(oldJob.stateName, _allowReset=False)
             except Exception:
                 printExc("Error occurred while trying to reset state from a previous error:")
-            raise exc[0], exc[1], exc[2]
+            raise exc[0](exc[1]).with_traceback(exc[2])
```

Comment:
I'm having a hard time remembering why I didn't just `raise exc[1]` here..

Reply:
In any case, I don't think `with_traceback` is backward compatible.

Reply:
maybe `six.reraise` ?

Reply:
sixed.